examples/adjoints/adjoints_ex5/general.in

```diff
@@ -20,7 +20,7 @@ deltat = 0.1
 # Make this 0 to use a constant deltat
 timesolver_tolerance = 0.0
 timesolver_upper_tolerance = 0.0
-timesolver_maxgrowth = 1.09
+timesolver_maxgrowth = 1.12
```

Comment:
Why the change here?  I know we have to change the gold standard results anyway since we're doing the integration more accurately, but it'd be nice to avoid an apples-to-oranges difference if we can at least get apples-to-crabapples.

Reply:
See above, the extra dt was making me think we reached 1.00 when we had only reached 0.9, so kept the number of timesteps the same, increased the size of the steps we are taking.

Reply:
Okay.  So can this be reverted then?